include/RAJA/pattern/detail/reduce.hpp

```diff
@@ -120,7 +120,7 @@ class ValueLoc
   constexpr ValueLoc() = default;
   constexpr ValueLoc(ValueLoc const &) = default;
 
-  RAJA_HOST_DEVICE ValueLoc &operator=(ValueLoc const &) = default;
+  ValueLoc& operator=(ValueLoc const &) = default;
```

Comment:
I think this change has broken stuff (see Travis).

Reply:
Yeah, it removes warnings with newer nvcc versions, but breaks older ones (like what we use on travis). I'll revert this.